Cargo.toml

```diff
@@ -54,6 +54,10 @@ windows = { version = "0.48.0", features = [
 bluer = { version = "0.16.1", features = ["bluetoothd"] }
 tokio = { version = "1.20.1", features = ["rt-multi-thread"] }
 
+[target.'cfg(target_os = "android")'.dependencies]
+jni-glue = { git = "https://github.com/akiles/jni-bindgen", rev = "45ddf3129bbe5987ab19f9c9be091de3bf3c5c25" }
```

Comment:
I'd like to keep `main` releasable on crates.io, so perhaps we should merge into an `android` branch until you're ready to publish `jni-glue`?

Reply:
I can publish the fork right now before merging. I think it's mostly ready (or at least ready to the extent `bluest` needs to work...). I just need to come up with a name :D

Reply:
i've released :spaghetti: [`java-spaghetti`](https://github.com/Dirbaio/java-spaghetti) v0.1.0, there's no git deps anymore.